### PR TITLE
Add TravisCI implementation

### PR DESCRIPTION
**Pull request by RyanSchw**
Use `pycodestyle` (which is also used locally) to check before any merges are made.

### CODE REVIEW

**Review by AdamBusch — DISMISSED** [review]

**Review by bsmrdel — DISMISSED** [review]

File: WirelessTelemServer.py [hunk 1]
```diff
@@ -7,49 +7,57 @@
...
-    print(f'Connected to client at {connection_addr} on {datetime.datetime.now()}.')
+    connection_addr = (f'{websocket.remote_address[0]}:' +
+                       f'{websocket.remote_address[1]}')
 
-    await websocket.send(json.dumps({'type':'connected','payload':'true', 'timestamp':time.time()}))
-    
+    print(f'Connected to client at {connection_addr}' +
+          f' on {datetime.datetime.now()}.')
+
+    await websocket.send(json.dumps({'type': 'connected', 'payload': 'true',
```
Comment (RyanSchw) [inline WirelessTelemServer.py]: https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas
If you try this does pycodestyle say you shouldn't have trailing commas?

**Comment by RyanSchw** [comment]
@josh05201997 When you get the chance, can you look over this? Main file to note is `.travis.yml`, as that specifies that any pull request must now pass the `pycodestyle` check before it can be merged.

**Review by AdamBusch — APPROVED** [review]
looks good!

**Review by joshshao — APPROVED** [review]

**Comment by RyanSchw** [comment]
Is there something you want to change @josh05201997 ?

**Comment by josh05201997** [comment]
yea, i just approved the thing, sorry i clicked the wrong thing

**Pull request merged**